### PR TITLE
SISRP-17974 - Update front-end dependencies - Follow up

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.berkeley.edu/browse/SISRP-17974

* Use the same version as on Bamboo
* Make sure Travis installs the correct version too

Follow up from #5253

### CODE REVIEW

**Comment by redconfetti** [comment]
@christianvuerings - Nice try. Is `0.12.4` the newest version that IS&T can support on the Bamboo server? That's pretty far behind given that v4.0 is stable right?

**Comment by christianvuerings** [comment]
@redconfetti https://jira.ets.berkeley.edu/jira/browse/CS-5363 is the upgrade jira

File: script/front-end-test.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
-# . $HOME/.nvm/nvm.sh
-# nvm install node
+if [ -d "$HOME/.nvm" ]; then
```
Comment (redconfetti) [inline script/front-end-test.sh]: 👍

**Comment by christianvuerings** [comment]
Going to make an update to only make `.nvm` conditional - since the bamboo update happened (thanks to @jon_felder)

**Comment by christianvuerings** [comment]
Looks like Travis is doing the right thing now:

```
$ ./script/front-end-test.sh
######################################################################## 100.0%
Checksums empty
Now using node v4.4.3
```

https://api.travis-ci.org/jobs/124305629/log.txt?deansi=true

**Comment by redconfetti** [comment]
Thanks @jon_felder and @christianvuerings 
🍰  🍔  🍨

**Pull request merged**